overlay.nix

```diff
@@ -0,0 +1,35 @@
+# TODO: move these overrides into a nixpkgs config option
+final: prev: {
+  cudatoolkit = prev.cudatoolkit_11;
```

Comment:
So if I want to add eg `tensorflowWithCuda` or `jaxlibWithCuda`, or will they be built anyways since this is an overlay? How does one see the CI status/results?

Reply:
Those should be added in the `extraChecks` list below. We're building all the unfree+redistributable packages, plus the ones in the `extraChecks` list.

Reply:
See https://github.com/numtide/nixpkgs-unfree/pull/8/files#diff-f838dd30f7aebd72dc21b473cc73695d1bc6c7eef3a3b32217ee163564f866b9R27

Reply:
gotcha! is `pkgs.config.cudaSupport` set to `true` somewhere as well?

Reply:
yes over here: https://github.com/numtide/nixpkgs-unfree/pull/8/files#diff-245563af9e34754739b12b294f3324c1d42c116ace7ce8735e21dfbc82d8e064R7